rabbitmq/tests/test_unit.py

```diff
@@ -173,3 +181,46 @@ def test_disable_nodes(aggregator, check):
     # check to ensure other metrics are being collected
     for m in metrics.Q_METRICS:
         aggregator.assert_metric(m, count=1)
+
+
+def test_queues_regexes_exclude_with_negative_lookahead(aggregator, dd_run_check):
+    """Based on a support case where a customer was confused why their regular expression didn't work.
+
+    The key piece optionally matching the vhost part.
+    """
+    data = {}
+    for ep in ("queues", "overview"):
+        with open("tests/fixtures/mgmt/{}.json".format(ep)) as fh:
+            data[common.URL + ep] = json.load(fh)
+
+    def mock_get_data(_self, url):
+        return data.get(url, [])
+
+    instance = {
+        "rabbitmq_api_url": common.URL,
+        "rabbitmq_user": "guest",
+        "rabbitmq_pass": "guest",
+        "queues_regexes": [
```

Comment:
I'm a bit confused about the value of `queues_regexes` here. Do you just have 1 big regex string?

Reply:
That's right! With comments too :)

All thanks to [this flag](https://docs.python.org/3/library/re.html#re.VERBOSE)